src/ert/shared/cli/model_factory.py

```diff
@@ -9,6 +10,15 @@
 
 
 def create_model(ert, ensemble_size, current_case_name, args, id_):
+    logger = logging.getLogger(__name__)
+    logger.info(
+        "Initiating experiment",
+        extra={
+            "mode": args.mode,
+            "ensemble_size": ensemble_size,
+        },
```

Comment:
Could we add all the args as well? Might be useful to have :thinking:

Reply:
short answer: no...